cmd/renter.go

```diff
@@ -177,8 +176,21 @@ func runRenterInit(args ...string) {
 	// Fetch user information from or register with metaserver
 	metaService := metaserver.NewClient(config.MetaAddr, &http.Client{})
 	if len(*keyFileFlag) > 0 {
-		// TODO: implement key import
-		log.Fatal("key import not implemented")
+		renterId := util.FingerprintKey(publicKeyBytes)
+		err := metaService.AuthorizeRenter(rsaKey, renterId)
+		if err != nil {
+			os.Remove(homeDir)
```

Comment:
any reason you're using Remove instead of RemoveAll?

Reply:
Nope. Autocomplete bug. Thanks for mentioning. Fixed in new commit.